docs/sources/setup-grafana/configure-security/configure-authentication/saml/index.md

```diff
@@ -373,6 +373,20 @@ SAML's single logout feature allows users to log out from all applications assoc
 `HTTP-Redirect` and `HTTP-POST` bindings are supported for single logout.
 When using `HTTP-Redirect` bindings the query should include a request signature.
 
+#### Configure single logout
+
+{{% admonition type="note" %}}
+Proper `NameID` and `SessionIndex` support is available in public preview behind the `improvedExternalSessionHandling` feature toggle, starting from Grafana v11.5.
```

Comment:
It is a bit confusing to read "proper `NameID", as it is unclear what is proper. I'd probably suggest first to have a small section on it's own describing the changes with `NameID` and `SessionIndex`, even before the `Configure single logout` section. And once this is done, we can simply change this note to say something like:

```
{{% admonition type="note" %}}
The improvedExternalSessionHandling feature toggle is available in public preview starting from Grafana v11.5. To enable it, refer to Feature Toggles Documentation.
{{% /admonition %}}
```

The section above this, can be something like

```
#### `NameID` and `SessionIndex` changes in Grafana v11.5

Before Grafana v11.5:
- Grafana used the Login attribute (from `assertion_attribute_login`) as the `NameID` in logout requests. This often caused issues if the `Login` value did not match the IdP’s expectations.
- Grafana did not support the `SessionIndex`, leading to potential issues like logging out all IdP sessions when logging out of Grafana.

Starting from Grafana v11.5
- Grafana uses the NameID from the SAML assertion for logout requests. If NameID is missing, Grafana defaults to the Login attribute.
- Grafana now supports including the SessionIndex in logout requests if it is provided by the IdP.
```

Does not have to be this exact format, but I hope I could explain the idea.